src/components/GenerateGifForm.js

```diff
@@ -21,15 +28,50 @@ class GenerateGifForm extends Component {
       case 'fontColor':
         this.props.updateTextColor(evt.target.value);
         break;
+
       case 'name':
         this.props.updateGIFFileName(evt.target.value);
         break;
+
+      case 'placement':
+        let values = evt.target.value.split('-');
+        let positionObj = {
+          textAlign: values[1],
+          textBaseline: values[0]
+        };
+        this.props.updateTextPosition(positionObj);
+        let selectValue = document.getElementById('GenerateGifForm-select');
+        selectValue = selectValue.options[selectValue.selectedIndex].value;
+        localStorage.setItem('selectValue', selectValue);
+        this.props.updateTextPosition(positionObj);
+
+        break;
       default:
         break;
     }
   }
 
+  renderColorPicker() {
+    this.setState(state => ({
```

Comment:
I'd prefer 'prevState' over 'state' just for clarity.

Reply:
Done